hello/hello.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+
+int main(int argc, char** argv) {
+  char* name = argc >= 2 ? argv[1] : "anonymous";
+
+  FILE* f_out = fopen("out.txt", "w");
```

Comment:
Hmm, you don't seem to check whether `fopen` was actually successful. In case that there is an error creating the output file for whatever reason, the error in the logs will be very cryptic. I suggest we add some minimal error checking here to avoid this problem.

I guess something along the lines of should work:

```c
if (f_out == NULL) {
    printf("couldn't open file for writing\n");
    return 1;
}
```